lib/trailblazer/operation/public_call.rb

```diff
@@ -36,8 +43,16 @@ def call_with_public_interface(*args)
     end
 
     # This interface is used for all nested OPs (and the outer-most, too).
-    def call_with_circuit_interface(args, circuit_options)
-      strategy_call(args, circuit_options) # FastTrack#call
+    #
+    # @param [Array] args - Contains [ctx, flow_options]
+    # @param [Hash]  circuit_options - Options to configure activity circuit
+    #
+    # @return [signal, [ctx, flow_options]]
+    #
+    # @private
+    def call_with_circuit_interface((options, flow_options), **circuit_options)
+      ctx = options_for_public_call(options, original_flow_options(flow_options))
+      strategy_call([ctx, original_flow_options(flow_options)], circuit_options) # FastTrack#call
```

Comment:
I'm pretty sure in the original version, `#flow_options` wasn't called for circuit interface, only for public interface! :metal:

Reply:
Ah hmm, but we want to pass `flow_options` given as arguments ahead, don't we ?

Reply:
Reverted this change.